src/global/Layouts/StickyNavigation/NavigationElement/NavigationElement.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Icon } from 'components/Icon';
+import { Col } from 'antd';
+import { Link } from 'react-router-dom';
+
+type NavActive = {
+  isActive: boolean;
+};
+
+const NavIcon = styled(Icon)<NavActive>`
+  background-color: ${props =>
+    props.isActive ? props.theme.colors.main.primary : props.theme.colors.functional.disabled};
+`;
+
+const CenteredCol = styled(Col)`
+  && {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+`;
+
+const NavText = styled.span.attrs({
+  style: (props: any) => props.theme.typography.subtitle2,
+})<NavActive>`
+  color: ${props => (props.isActive ? props.theme.colors.main.primary : props.theme.colors.functional.disabled)};
+  display: ${props => !props.isActive && 'none'};
+`;
+
+const StyledLink = styled(Link)`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`;
+
+type NavigationElementType = {
```

Comment:
Maybe let's keep types definitions next to each other?

Reply:
Changed.